docs/anonymizer/adding_operators.md

```diff
@@ -0,0 +1,18 @@
+# Supporting new types of PII operators
+
+Presidio anonymizer can be easily extended to support additional anonnymization and deanonymization methods.
+
+## Extending the anonymizer for additional PII operators:
```

Comment:
```suggestion
## Extending presidio-anonymizer for additional PII operators:
```
This might be confusing as we have both anonymization and deanonymization now.